bundles/org.openhab.binding.weathercompany/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,523 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="weathercompany" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+		<label>Weather Company Bridge</label>
```

Comment:
```suggestion
		<label>Weather Company Account</label>
```

Reply:
Of course. Sometimes you can't see the forest for the trees... 🙄